app/models/miq_expression.rb

```diff
@@ -1007,7 +992,7 @@ def self.get_col_info(field, options = {})
   def self.exclude_col_by_preprocess_options?(col, options)
     return false unless options.kind_of?(Hash)
     return false unless options[:vim_performance_daily_adhoc]
-    EXCLUDED_COLS_FOR_EXPRESSIONS.include?(col.to_sym)
+    Metric::Rollup.excluded_col_for_expression?(col.to_sym)
```

Comment:
I see, so MetricRollup still connects to the database, but we're no longer loading it from ApplicationController -> MiqExpression